UIColor+hexValue.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  UIColor+hexValue.swift
+//  Rocket.Chat.iOS
+//
+//  Created by Kornelakis Michael on 9/2/15.
+//  Copyright © 2015 Rocket.Chat. All rights reserved.
+//
+
+import UIKit
+
+
+extension UIColor {
+    
+    static func colorWithHexValue(redValue: CGFloat, greenValue: CGFloat, blueValue: CGFloat, alpha: CGFloat) -> UIColor {
```

Comment:
You're already using some colors into the code... using that colors is a great start! :+1: